themes/startwords/README.md

```diff
@@ -1,31 +1,28 @@
 
-[Live Demo](https://themes.gohugo.io/theme/blank/)
+The Startwords Hugo theme is designed for [a journal of the same name](https://startwords.cdh.princeton.edu/) by the [Center for Digital Humanities](cdh.princeton.edu/) at Princeton University.
 
-![Blank theme screenshot](https://github.com/Vimux/blank/blob/master/images/splash.png)
+## Goals
 
-## Installation
+The main architectural goals of this theme are to:
+- reduce complexity and focus on the content by using a static site generator
+- support a range of viewing devices (including mobile phones, tablets, laptops) and output formats (including html, plain text, and PDF)
+- use automation & tools to reduce manual steps in publishing workflow
+- apply innovative, thought-provoking designs to a digital publication
 
-In your Hugo site `themes` directory, run:
+## Features
 
-```
-git clone https://github.com/vimux/blank
-```
+- Multiple article output formats: articles are generated as .txt files using Hugo's `{{ .Plain }}` [page variable](https://gohugo.io/variables/page/) and as PDFs using [WeasyPrint](https://weasyprint.org/)
+- Layouts: custom content types include feature articles [available under [layouts/article](https://github.com/Princeton-CDH/startwords/tree/master/themes/startwords/layouts/article)], single issues, and list of issues [both available under [layouts/issue](https://github.com/Princeton-CDH/startwords/tree/master/themes/startwords/layouts/issue)].
+- To retain the simplest directory structure that will give us the URLs we want, articles are placed in an issue number directory and have `type:article` specified in the page metadata.
+- Excerpts of the current issue's opening lines are generated on the homepage using Hugo's `<!--more-->` [content summary divider](https://gohugo.io/content-management/summaries/)
+- Article DOIs (registered with [Zenodo](zenodo.org/)) are specified in each article's YAML header, so that article metadata can be easily harvested by [Zotero](https://www.zotero.org/) citation management software.
+- Article footnotes are generated as **contextual notes** using…
+- Article illustration capabilities are built in using [IIIF](https://iiif.io/) for images and [Sketchfab](https://sketchfab.com/) for 3D embeddings.
 
-Next, open `config.toml` in the base of the Hugo site and ensure the theme option is set to `blank`.
+## Information Architecture
 
-```
-theme = "blank"
-```
-
-For more information read the official [quick start guide](https://gohugo.io/getting-started/quick-start/) of Hugo.
-
-## Contributing
-
-Have you found a bug or got an idea for a new feature? Feel free to use the [issue tracker](https://github.com/Vimux/blank/issues) to let me know. Or make directly a [pull request](https://github.com/Vimux/blank/pulls).
-
-## License
-
-This theme is released under the [MIT license](https://github.com/Vimux/blank/blob/master/LICENSE).
+![](https://startwords.cdh.princeton.edu/Information%20Architecture.svg)
```

Comment:
I was going to ask if the architecture diagram belongs here, but I looked at it again and it does seem like our site architecture is pretty important for understanding the content types and the way we've set them up.